book/website/community-handbook/newsletters/newsletters-process.md

```diff
@@ -0,0 +1,113 @@
+# Process of Creating a newsletter
+
+This document provides an overview of a process we use in _The Turing Way_ for drafting, reviewing and publishing newsletters.
+Though written for _The Turing way_, these steps can be adapted for documenting newsletter for any project.
+
+We only suggest using these recommendations as guides.
+These should not be considered as a set of fixed rules or the "only" way one should create newsletters.
+As individual authors, you should allow your personality to show in your newsletter drafts.
+After all, these newsletters land in someone's personal mailbox and most likely they know you (or your community).
+
+## Steps for drafting _The Turing Way_ newsletters
+
+### Create a GitHub issue to collect items
+
+Create a new GitHub issue where throughout the month you and other community members can suggest news items as a comment.
+Link to the issue for next month can be published in your most recent newsletter.
```

Comment:
This sentence is missing something, might just be that it needs to be "a link to"?

Reply:
Good point. I can add an example link from the past month. For example: https://github.com/alan-turing-institute/the-turing-way/issues/1037?

Reply:
I have added suggestion in your comment with the link to the issue and why it is being created. Does this look alright?

Reply:
Looks good to me 😄